src/components/panel/panel.js

```diff
@@ -806,6 +850,75 @@ MdPanelService.prototype.newPanelAnimation = function() {
 
 
 /**
+ * Creates a panel group and adds it to a tracked list of panel groups.
+ * @param groupName {string} Name of the group to create.
+ * @param config (!Object=) Specific configuration object that may contain the
+ *     following properties:
+ *
+ *   - `maxOpen` - `{number=}`: The maximum number of panels that are allowed
+ *     open within a defined panel group.
+ */
+MdPanelService.prototype.newPanelGroup = function(groupName, config) {
+  if (!this._groups[groupName]) {
```

Comment:
I think that it might be better to use `hasOwnProperty` here. This might cause undefined behaviour in the edge case if somebody names their group `toString` or something similar.